### DISCUSSION TITLE
[Feature] Android app: Trusted Certification Authorities

### DISCUSSION THREAD

**Discussion by BestBackwards** [immich-app/immich#8602] [Feature Request]
### The feature

When running Immich with a private ceritification authority, rather than a public one, the app does not trust the leaf certificate despite the signing CA certificate being in the trusted root store. This means having to turn off certificate validation completely, weakening the security.

Please ensure that the app performs certificate validation against all trusted root CAs on the device.

I am not sure if this affects iOS also.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2139

**Reply by BestBackwards** [reply]
My bad
